lib/PuppeteerSharp/DevToolsTracing.cs

```diff
@@ -23,7 +23,7 @@ namespace PuppeteerSharp
     /// await Page.Tracing.StopAsync();
     /// </code>
     /// </example>
-    public class Tracing
+    public class DevToolsTracing
```

Comment:
This is a breaking change. Do we need to do this?

Reply:
The CA1724 rule was emitting an error for the class name Tracing. However, it looks like keeping parity with puppeteer class names is important, so I think the best approach may be to suppress the error for this class, rather than renaming it. I should have those changes for review soon.